pydantic/main.py

```diff
@@ -374,6 +373,8 @@ def is_untouched(v: Any) -> bool:
         cls = super().__new__(mcs, name, bases, new_namespace, **kwargs)
         # set __signature__ attr only for model class, but not for its instances
         cls.__signature__ = ClassAttribute('__signature__', generate_model_signature(cls.__init__, fields, config))
+        cls.__try_update_forward_refs__()
```

Comment:
What is the impact on perf for models without forward refs? Do we have a way to know whether or not we should try to update?

Reply:
I wrote a simple benchmark:
```py
import timeit
from pydantic import BaseModel


def callback():
    class Order(BaseModel):
        a: int
        b: float
        c: str


t = timeit.timeit(
    callback,
    number=10_000,
    globals={"BaseModel": BaseModel},
)

print(f'{t:.5f}')
```

Results:
* With `__try_update_forward_refs__` - 3.17182
* Without `__try_update_forward_refs__` - 3.15772